src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -20,7 +20,7 @@ const AuthenticatorRow = View.extend({
   template: hbs`
     <div class="authenticator-icon-container">
       {{#if logoUri}}
-        <div class="factor-icon authenticator-icon custom-app-logo" role="img"
+        <div class="factor-icon authenticator-icon {{iconClassName}} custom-logo" role="img"
```

Comment:
I think the better option here would be to add a another condition to this `#if logoUri` because this section is specifically for the Custom App push authenticator. You could either make the property `customLogoUri` instead of `logoUri` and then add the additional if check on that property (along with the corresponding changes in the AuthenticatorUtil file). 

We can discuss this during office hours if needed.

Reply:
I based this refactoring on `AuthenticatorVerifyOptions.js`, which already has this code logic (you can see the refactor [here](https://github.com/okta/okta-signin-widget/commit/d41b033a33a9c35014efbe82f9c00cc4dee3af6c#diff-bcd9127a5716d292050e226cf7d456d7c3befd3c354dce342dac6debf4eb9be0R20)).

Reply:
I'm confused as to what part of this section is specific to the Custom App push authenticator?